packages/protocol/contracts-0.8/common/SuperBridgeETHWrapper.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.15;
+
+import "../../contracts/common/Initializable.sol";
+import "@openzeppelin/contracts8/access/Ownable.sol";
+import "./interfaces/IStandardBridge.sol";
+import "./interfaces/IWETH.sol";
+
+contract SuperBridgeETHWrapper is Initializable, Ownable {
+  uint32 internal constant DEFAULT_GAS_LIMIT = 200_000;
```

Comment:
Is default gas limit enough or should we let the user pass the limit to function wrapAndBridge?